### PR TITLE
Initial Structure for review

### PR DESCRIPTION
**Pull request by jkowalcz**
@PharkMillups @kent-williams 

For your guidance, this would be a framework  to document the miner CLI

### CODE REVIEW

**Comment by jkowalcz** [comment]
> @PharkMillups @kent-williams
> 
> For your guidance, this would be a framework to document the miner CLI

So I have the basic structure I'm thinking and peer mostly drafted out

**Comment by kent-williams** [comment]
@jkowalcz this is looking like a good start! So first major change that feels needed is to add an additional subsection for this since it's not the only CLI. So instead of `/cli/` as the root we change it to `/cli/miner` .

File: docs/cli/minercli.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
+---
+id: minercli
+sidebar_label: miner CLI
```
Comment (kent-williams) [inline docs/cli/minercli.mdx]: Can we capitalize miner here to follow the standard practice for the docs.

Comment (jkowalcz) [reply docs/cli/minercli.mdx]: Done.

**Comment by kent-williams** [comment]
Apologies, let's instead put it under `/mine-hnt/miner-cli/`, and make the sidebar label `CLI`, like we've done with https://deploy-preview-604--helium-docs.netlify.app/wallets/cli-wallet.

File: docs/mine-hnt/miner-cli/minercli.mdx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+id: minercli
+sidebar_label: Miner CLI
+hide_title: true
+slug: /mine-hnt/miner-cli/minercli
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+---
```
Comment (PharkMillups) [inline docs/mine-hnt/miner-cli/minercli.mdx]:
```suggestion
```

**Comment by PharkMillups** [comment]
@jkowalcz Awesome work here. A few comments:

* We need an introduction. Can we add a few words at the top of the main page that explains what the miner CLI is and what it's used for? 
* Also in that intro, let's get a "Note" that points to the source code on GitHub.
* We should have a call out (probably in that same intro section) that points to where on discord to get help using it. `#blockchain-development` is probably the best.
* It would be good to describe the structure somewhere. E.g. "For each General command below, all the available commands are listed. There is also a dedicated sub page per command linked at the top of each section that goes into detail on the command output."
* Bit nit picky, but on heading structure, I would go with `#` for the intro piece, use `##` for "Top Level Commands" and "General Commands" and `###` for all the individual command sections.

File: docs/mine-hnt/miner-cli/minercli.mdx (hunk 2 above)
Comment (jkowalcz) [reply docs/mine-hnt/miner-cli/minercli.mdx]: Updated.

File: docs/mine-hnt/miner-cli/minercli.mdx [hunk 3]
```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
----
+# Introduction
```
Comment (PharkMillups) [inline docs/mine-hnt/miner-cli/minercli.mdx:11]: I think you need a line break below (and maybe above) this for markdown to pick it up.

File: docs/mine-hnt/miner-cli/minercli.mdx [hunk 4]
```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
...
+The `miner` command controls validators and hotspot miners it is also the core software that runs all aspects of the blockchain, like adding txns, and validating blocks while in a consensus group. The CLI is used to troubleshoot and maintain these devices. 
+
+You can always get more detailed help in the `#blockchain-development` channel in [Discord](https://discord.com/channels/404106811252408320/730245219974381708).
+
+:::info
+
+The github repository to review the code for the miner command is found [here](https://github.com/helium/miner) Note the differing tags for validators and hotspots.
```
Comment (PharkMillups) [inline docs/mine-hnt/miner-cli/minercli.mdx]: Add "." after miner link.

File: docs/mine-hnt/miner-cli/minercli.mdx [hunk 5]
```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
----
+# Introduction
+The `miner` command controls validators and hotspot miners it is also the core software that runs all aspects of the blockchain, like adding txns, and validating blocks while in a consensus group. The CLI is used to troubleshoot and maintain these devices.
```
Comment (PharkMillups) [inline docs/mine-hnt/miner-cli/minercli.mdx]: "The `miner` command controls validators and hotspot miners..." ->
"The `miner` command line interface (CLI) is a tool to administer and interrogate Helium Validators and Hotspots, both of which run the core `miner` software."

File: docs/mine-hnt/miner-cli/minercli.mdx [hunk 6]
```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
...
+
+:::info
+
+The github repository to review the code for the miner command is found [here](https://github.com/helium/miner) Note the differing tags for validators and hotspots.
+
+:::
+
+## Example page syntax
+
+The complexity of all the different branches of the command is located on sperate pages. One for each major section of the command to keep the page length small. A navgation menu will be on the right hand column to navigate through each sub command. There is a Tab for the Command "inputs" and example outputs of each command are also on an individual tab.
```
Comment (PharkMillups) [inline docs/mine-hnt/miner-cli/minercli.mdx]: "The complexity of all the different branches of the command is located on sperate pages. One...small" -> 
"The full usage details for all `miner` commands are located on dedicated command pages. There is one for each top-level command to keep the docs focused."

**Comment by PharkMillups** [comment]
Excellent. Left a few more comments. Not sure why GitHub wasn't giving me the ability to leave suggestions in-line.

File: docs/mine-hnt/miner-cli/minercli.mdx (hunk 3 above)
Comment (jkowalcz) [reply docs/mine-hnt/miner-cli/minercli.mdx:11]: OK, weirdness is when in my local Yarn it works but the online preview doesn't work. 

Fixed it.

File: docs/mine-hnt/miner-cli/minercli.mdx (hunk 4 above)
Comment (jkowalcz) [reply docs/mine-hnt/miner-cli/minercli.mdx]: done

File: docs/mine-hnt/miner-cli/minercli.mdx (hunk 5 above)
Comment (jkowalcz) [reply docs/mine-hnt/miner-cli/minercli.mdx]: This was a little unclear hopefully this is what you were going for.

File: docs/mine-hnt/miner-cli/minercli.mdx (hunk 6 above)
Comment (jkowalcz) [reply docs/mine-hnt/miner-cli/minercli.mdx]: fixed

**Comment by PharkMillups** [comment]
This is looking great. Still can't that that top main title to show up, though. Otherwise I think we are just about there. @kent-williams What do you think?

File: docs/mine-hnt/miner-cli/authorize.mdx [hunk 7]
```diff
@@ -0,0 +1,186 @@
+---
+id: minercli-authorize
+sidebar_label: authorize
+hide_title: true
```
Comment (kent-williams) [inline docs/mine-hnt/miner-cli/authorize.mdx:4]: Let's make this false or remove it entirely on every doc in this PR, we want the titles to render.

File: sidebarsDocs.js [hunk 8]
```diff
@@ -93,6 +94,15 @@ module.exports = {
     },
     'api/console',
   ],
+  minercli: [
+    'mine-hnt/miner-cli/minercli',
```
Comment (kent-williams) [inline sidebarsDocs.js:104]: When we create a nested menu like you have, we usually make the link at the top a back button, just replace this line with below to do so.

```
{
      type: 'link',
      label: '<- Back',
      href: ''mine-hnt/miner-cli/minercli'
},
```

**Comment by jkowalcz** [comment]
@PharkMillups This can probably get closed in deference to #874

**Comment by PharkMillups** [comment]
Closing per author's request in #874

**Pull request closed without merging**